.github/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 8. Enhancement - Added Sumburgh (EGPB) online rings - thanks to @hsugden (Harry Sugden)
 9. Enhancement - Added ability to Observe SCO_CTR w/out STC, for when mentoring is split between SCO/STC thanks to @hsugden (Harry Sugden)
 10. Enhancement - Added various bandbox/split possibilities for ScAC and STC - thanks to @hsugden (Harry Sugden)
+11. AIRAC (1909) - EGTE SMR update - thanks @inventor02 (George Peppard)
```

Comment:
```suggestion
11. AIRAC (1909) - Added Exeter (EGTE) SMR holding point labels - thanks @inventor02 (George Peppard)
```

Just makes it clearer for Chris/members when this changelog is turned into the forum version as to what's changed! 😄